app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  
+  def new
+    
+  end
+  
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
```

Comment:
`params[:session][:email].downcase` Try to use hash dig function to avoid null pointer exception